lib/ide/parser/src/main.rs

```diff
@@ -8,7 +8,7 @@ fn main() {
     println!("Will parse: {}", program);
 
     let mut parser = parser::Parser::new_or_panic();
-    let     output = parser.parse(program);
+    let     output = parser.parse(program, vec![]);
```

Comment:
in place of `vec![]` I'd rather use `default()` -- this code should not need to commit to any specific container storage used by idmap.